CONTRIBUTING.md

```diff
@@ -57,6 +57,18 @@ $ git config user.name "My Name"
 $ git config user.email "my-email@example.com"
 ```
 
+> Note: if you forget to use the right email address the first time, and
```

Comment:
maybe change this to "the right combination of name and email address" to make it 100% clear that either can be fixed with `--reset-author` assuming janusgraph-bot is checking for the right combination of fields corresponding to the CLA and not just an email address that signed the CLA and is associated to the account submitting the PR.